examples/step-47/doc/results.dox

```diff
@@ -246,7 +247,7 @@ make sense:
   -\sum_{K \in \mathbb{T}} \int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}) = -\sum_{e \in \mathbb{F}^i} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}})
   - \sum_{e \in \partial \Omega} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} h,
   @f}
-  Where $\mathbb{F}^i$ is the set of interior faces.
+  where $\mathbb{F}^i$ is the set of interior faces.
```

Comment:
We use $\mathbb{F}^b$ in the introduction section as boundary interfaces but don't mention interior interfaces. So I set interior faces to be $\mathbb{F}^i$.